docs/index.md

```diff
@@ -27,7 +27,7 @@ ARGUMENTS
                                Use either this or the dataset-id argument
      -l, --log-file  <arg>     The name of the logfile in csv format. If not provided a file
                                easy-fedora2vault-<timestamp>.csv will be created in the home-dir of the user.
-                               (default = /home/vagrant/easy-fedora2vault-2020-02-02T20:20:02.000Z.csv)
+                               (default = /home/vagrant/easy-fedora2vault-2020-02-02T20-20-02-000Z.csv)
```

Comment:
I didn't like the `:` in a file name. Any other opinions?

Reply:
How about `_`? With `-` it looks too similar to date separator.

Reply:
On the other hand, the CSV report created by `easy-split-multi-deposit` also uses `:`

Reply:
i'd rather have it look like a datetime, so with the `:` in it